pkg/utils/kubeconfig/kubeconfig_test.go

```diff
@@ -225,3 +274,71 @@ users:
 func writeConfig(filename string) error {
 	return ioutil.WriteFile(filename, []byte(minikubeSample), os.FileMode(0755))
 }
+
+// If tempKubeconfigPath is a valid file path, sets the KUBECONFIG env variable to the file path
+// provided by tempKubeconfigPath and returns a function that restores KUBECONFIG to its previous
+// value and deletes the file at tempKubeconfigPath.
+// Returns an error if tempKubeconfigPath is not a valid file path.
+func manageConfigFile(tempKubeconfigPath string) (func(), error) {
+	if _, err := os.Stat(tempKubeconfigPath); os.IsNotExist(err) {
+		return nil, err
+	}
+	kubeconfigPathToRestoreAtEnd, hasKubeconfigPathToRestoreAtEnd := os.LookupEnv("KUBECONFIG")
+	os.Setenv("KUBECONFIG", tempKubeconfigPath)
+
+	return func() {
+		if hasKubeconfigPathToRestoreAtEnd {
+			os.Setenv("KUBECONFIG", kubeconfigPathToRestoreAtEnd)
+		} else {
+			os.Unsetenv("KUBECONFIG")
+		}
+		os.Remove(tempKubeconfigPath)
+	}, nil
+}
+
+// Cluster names are 'cluster-one.us-west-2.eksctl.io' and 'cluster-two.us-west-2.eksctl.io'.
+// All the information for cluster-one.us-west-2.eksctl.io is identical to one-cluster.yaml. If all
+// the information for cluster-two.us-west-2.eksctl.io is deleted, the file should be identical to
+// one-cluster.yaml and oneClusterAsBytes.
+var twoClustersAsBytes, _ = ioutil.ReadFile("./two-clusters.yaml")
```

Comment:
With the move of the test files:

`ioutil.ReadFile("./testdata/two-clusters.yaml")`

Reply:
Oh wow 😱 thanks!